modules/osd-create-cluster-ccs.adoc

```diff
@@ -166,6 +166,23 @@ PVs created by using any other storage class are still encrypted, but the PVs ar
 After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type in a machine pool. The number and types of nodes available to you depend on your {product-title} subscription.
 ====
 
+. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances to use only IMDSv2. The two ways to access instance metadata from a running instance are:
```

Comment:
IBM SG suggests avoiding continuing sentences into a list from a lead-in. Maybe:
```suggestion
. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances to use only IMDSv2. You can access instance metadata from a running instance in two ways:
```
or
```suggestion
. Choose your preference for the Instance Metadata Service (IMDS) type, either using both IMDSv1 and IMDSv2 types or requiring your EC2 instances to use only IMDSv2. There are two methods for accessing instance metadata from a running instance:
```